src/main/java/nextstep/subway/application/LineService.java

```diff
@@ -0,0 +1,57 @@
+package nextstep.subway.application;
+
+import nextstep.subway.domain.Line;
+import nextstep.subway.domain.LineRepository;
+import nextstep.subway.domain.Station;
+import nextstep.subway.dto.LineRequest;
+import nextstep.subway.dto.LineResponse;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.util.List;
+import java.util.NoSuchElementException;
+import java.util.stream.Collectors;
+
+@Service
+@Transactional(readOnly = true)
+public class LineService {
+    private LineRepository lineRepository;
+    private StationService stationService;
```

Comment:
final 을 붙여주면 어떨까요? 😄 
```suggestion
public class LineService {
    private final LineRepository lineRepository;
    private final StationService stationService;
```

Reply:
final 추가하였습니다!